.github/workflows/python.yml

```diff
@@ -54,7 +71,12 @@ jobs:
       - name: Install dependencies
         run: pip install -e .[dev]
       - name: Run tests
-        run: pytest tests
+        run: pytest --cov --cov-report=xml
```

Comment:
The tests are currently only run on 3.12, while 3.13 is also valid. Maybe add a test matrix and only compute coverage for one py version?